src/blocks/gallery-masonry/styles/editor.scss

```diff
@@ -18,118 +17,119 @@
 		}
 	}
 
-	// Fixes issue where figures are set to display 100% if alignfull is set.
-	.coblocks-gallery--figure {
-		width: auto !important;
+	.gallery-media-placeholder-wrapper {
+		margin-top: 20px;
 
-		&.is-selected {
-			border-radius: $radius-block-ui + 2; // Additional because we modify the selected state.
+		.components-placeholder__fieldset {
+			align-items: center;
 
-			&::after {
-				border-radius: $radius-block-ui + 2;
-				box-shadow: inset 0 0 0 $border-width + 1 var(--wp-admin-theme-color, #007cba), inset 0 0 0 3px $white;
+			button {
+				margin-bottom: 0;
 			}
 		}
-
-		img {
-			vertical-align: bottom;
-		}
 	}
+}
 
-	// Fixes issue with multiple dropzones displaying when dropping in an image.
-	.components-drop-zone.is-active {
-
-		+ ul li:last-child .components-drop-zone {
-			display: none;
-		}
+figure.wp-block-coblocks-gallery-masonry {
+	&.alignwide,
+	&.alignfull {
+		margin: 0;
 	}
 
-	// Captions.
-	.block-editor-rich-text {
-		bottom: 0;
-		left: 0;
-		max-height: 100%;
-		overflow-y: auto;
-		position: absolute;
-		right: 0;
-		width: 100%;
-
-		.editor-rich-text__tinymce {
-			color: inherit;
-
-			a {
-				color: inherit;
+	> .masonry-grid {
+		column-fill: initial;
+		column-gap: 30px;
+		transition: all 0.5s ease-in-out;
+
+		figure {
+			display: block;
+			margin-bottom: 30px;
+			margin-top: 0;
+
+			img {
+				border: 0;
+				display: block;
+				font: inherit;
+				font-size: 100%;
+				height: auto;
+				margin: 0;
+				max-width: 100%;
+				padding: 0;
+				width: 100%;
 			}
 
-			&:not(.mce-content-body) {
-				opacity: 0.6;
+			figcaption {
+				background: linear-gradient(0deg, rgba($color: $black, $alpha: 0.7) 0, rgba($color: $black, $alpha: 0.3) 50%, transparent);
+				bottom: 0;
+				font-size: 13px;
```

Comment:
We should try to use "em" or "rem" for font-sizes. It will scale better when zooming

Reply:
That sounds good to me. My only hesitation is that a quick scan of the code reveals that we set captions on most galleries to `font-size: 13px` including stacked and offset galleries. These are similar styles in that way. I wonder if we should change them all at once?